src/componenets/calculator.js

```diff
@@ -0,0 +1,30 @@
+function CreateStructure() {
```

Comment:
## Optional
- I suggest that you call this function `Calculator` to make it more meaningful
- Please remove all the boilerplate that got created by the CRA (texts, images, and styles) anything you are not using remove it
<img width="255" alt="Screenshot 2023-07-14 at 16 22 15" src="https://github.com/geekelo/recalc-webapp/assets/44745855/b128c2a1-63eb-4aeb-9b56-adb5661e837f">

Reply:
Thank you @DeliceLydia 
I have effected the optional changes you suggested.